modules/post/chrome/gather/cookies.rb

```diff
@@ -0,0 +1,176 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'net/http'
+require 'json'
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+
+  GET_ALL_COOKIES_REQUEST = '{"id": 1, "method": "Network.getAllCookies"}'.freeze
+  WEBSOCAT_URL = 'https://github.com/vi/websocat/releases/download/v1.2.0/websocat_nossl_'.freeze
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name' => 'Chrome Gather Cookies',
+                      'Description' => "
+                      Read all cookies from the Default Chrome profile of the target user. Downloads and executes https://github.com/vi/websocat to communicate with the remote debugging interface Chrome exposes, and writes to disk.
+                      ",
+                      'License' => MSF_LICENSE,
+                      'Author' => ['mangopdf <mangodotpdf[at]gmail.com>'],
+                      'Platform' => %w[linux unix bsd osx python],
+                      'SessionTypes' => %w[meterpreter shell]))
+
+    register_options(
+      [
+        OptString.new('CHROME_BINARY_PATH', [false, 'The path to the user\'s Chrome binary (leave blank to use the default for the OS)', '']),
```

Comment:
Double quotes are preferred over escaping.

```suggestion
        OptString.new('CHROME_BINARY_PATH', [false, "The path to the user's Chrome binary (leave blank to use the default for the OS)", '']),
```